doc/api/http2.md

```diff
@@ -1615,6 +1615,7 @@ added: v8.4.0
 
 The `'timeout'` event is emitted when there is no activity on the Server for
 a given number of milliseconds set using `http2server.setTimeout()`.
+**Default:** `2 Minutes`.
```

Comment:
I would remove the backticks here, those are typically used to enclose a class name or a literal (JS) value.

Reply:
Done.